app/services/trading_strategies.py

```
@@ -4153,6 +4153,405 @@ def _calculate_portfolio_sortino(self, positions: List[Dict]) -> float:
+
+    def _generate_leverage_adjustment_steps(self, action: str, current: float, target: float) -> List[Dict]:
+        """Generate steps for leverage adjustment."""
+        try:
+            steps = []
+            if action == "increase":
+                steps.append({"step": 1, "action": "Add margin to account"})
+                steps.append({"step": 2, "action": f"Increase leverage from {current}x to {target}x"})
+            elif action == "decrease":
+                steps.append({"step": 1, "action": f"Decrease leverage from {current}x to {target}x"})
+                steps.append({"step": 2, "action": "Withdraw excess margin"})
+            return steps
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)